web/components/widgets/linkify.tsx

```diff
@@ -38,7 +38,13 @@ export function Linkify(props: { text: string; className?: string }) {
     <span className={clsx(className, 'break-anywhere')}>
       {text.split(regex).map((part, i) => (
         <Fragment key={i}>
-          {part}
+          {/* Ensure newlines are respected */}
+          {part.split('\n').map((line, j) => (
```

Comment:
I think there's a css-only solution with whitespace-preline or something. Will take a look tomorrow

Reply:
You are right! Just pushed up a css-only version.